Sources/Plasma/FeatureLib/pfJournalBook/pfJournalBook.cpp

```diff
@@ -2210,7 +2210,7 @@ bool    pfJournalBook::IGetNextOption( const wchar_t *&string, wchar_t *name, wc
         return false;
 
     // Copy name
-    size_t len = ((uintptr_t)string - (uintptr_t)c)/2; // divide length by 2 because each character is two bytes
+    size_t len = ((uintptr_t)string - (uintptr_t)c)/sizeof(wchar_t);
```

Comment:
`string` and `c` are already `wchar_t` pointers...  This can (and should) be simplified to just:
```suggestion
    size_t len = string - c;
```

Reply:
That’s a really great point. I’ll make the change.